examples/single_instance/main.tf

```diff
@@ -0,0 +1,52 @@
+provider "aws" {
+  region = local.region
+}
+
+data "aws_availability_zones" "available" {}
+
+locals {
+  name   = "single-postgresql-instance"
+  region = "eu-central-1"
+
+  tags = {
+    Name = local.name
+  }
+}
+
+module "rds_instance_test" {
+  source                       = "../../"
+  environment                  = "dev"
+  identifier                   = local.name
+  is_kubernetes_app_enabled    = false
+  is_proxy_included            = false
+  service_availability         = "low"
+  username                     = "single_instance_user"
+  vpc_id                       = module.vpc.vpc_id
+  is_publicly_accessible       = true
+  subnet_ids                   = concat(module.vpc.public_subnets)
+  resource_owner_contact_email = "example@dfds.com"
```

Comment:
the postmaster at dfds will probably be happy to avoid scrapers spamming us. Perhaps we ough to use 
noreply@example.com since example.com is a domain used for examples?

Reply:
done